File: condicoes/1046.py
#1046 TEMPO DE JOGO
valores = input().split(" ")
HoraInicial = int(valores[0])
HoraFinal = int(valores[1])

if HoraInicial == HoraFinal:
    print("O JOGO DUROU 24 HORA(S)")
else:
    calculo = (24-(24+HoraInicial-HoraFinal)%24)
    print("O JOGO DUROU {} HORA(S)" .format(calculo))

File: javascript/1046.js
/*Tempo de Jogo*/
let input = require("fs").readFileSync("/dev/stdin", "utf8");

let lines = input.split(" ");

let Inicio = parseInt(lines.shift());
let Fim = parseInt(lines.shift());

let Hora = 24 - (24 + Inicio - Fim) %24;

if(Inicio == Fim){
  console.log("O JOGO DUROU 24 HORA(S)");
}else{
  console.log("O JOGO DUROU " + Hora + " HORA(S)");
}